src/main/java/uk/gov/gchq/palisade/util/DebugUtil.java

```diff
@@ -51,7 +51,7 @@ public static boolean checkDebugMode() {
     public static void updateDebugMode() {
         try {
             isDebug = Boolean.valueOf(System.getProperty(DEBUG, DEBUG_DEFAULT).trim());
-            if (isDebug) {
+            if (Boolean.TRUE.equals(isDebug)) {
```

Comment:
Boolean.valueOf is guaranteed to either throw an exception or return Boolean.TRUE or Boolean.FALSE.
if (isDebug) should call the unboxed value of isDebug - so it would call isDebug.booleanValue()
While Boolean.TRUE.equals(isDebug) would technically take less instructions - I think it's a lot harder to read.
Is there a good reason why this is better???

Reply:
using this modified if will also check for nulls as well as false, whereas the original will throw an exception if isDebug is ever null

Reply:
agreed that neither option is superior to the other so the easiest solution is to leave the change